components/ui/ace-editor.vue

```diff
@@ -111,6 +185,15 @@ export default {
       if (this.lint) this.provideLinting(content)
     })
 
```

Comment:
We have an editable JSON editor for the POST request JSON bodies. Can we get live updates for that ?
(Make sure to debounce)

Reply:
so you basically want outlining for the post request editor as well?

Reply:
The component used on the POST request editor is the same as the component used on the Ace Editor, just that it is writable. So the JSON path thingy already comes on top when it is edited.

Reply:
ive made it so that it updates the current path as you type now, the dropdown isnt yet visible though

Reply:
ok now it should work in the request editor too! @AndrewBastin

Reply:
It works, but it kinda messes up some times. I guess we will disable the outline for the request body.

Reply:
messes up in what sense? for me the only time it doesnt work is when the json itself is invalid, but cant really do anything about that. did you notice any other issues?

Reply:
When I tried it out, it struggled a bit to display the correct outline info sometimes, it lags behind sometimes.